### PR TITLE
Remove invisible redundant table cell

### PR DESCRIPTION
**Pull request by knightsifive**

### CODE REVIEW

**Comment by knightsifive** [comment]
Not sure what I changed on line 607 (see Github diff).

File: riscv-asm.md [hunk 1]
```diff
@@ -604,4 +604,4 @@ frflags rd        | csrrs rd, fflags, x0       | Read FP exception flags
 fsflags rd, rs    | csrrw rd, fflags, rs       | Swap FP exception flags
 fsflags rs        | csrrw x0, fflags, rs       | Write FP exception flags
 fsflagsi rd, imm  | csrrwi rd, fflags, imm     | Swap FP exception flags, immediate
-fsflagsi imm      | csrrwi x0, fflags, imm     | Write FP exception flags, immediate
\ No newline at end of file
+fsflagsi imm      | csrrwi x0, fflags, imm     | Write FP exception flags, immediate
```
Comment (knightsifive) [inline riscv-asm.md:607]: What did I change here? Strange...

Comment (knightsifive) [reply riscv-asm.md:607]: Perhaps it added a newline? I used the Github web editor.

Comment (knightsifive) [reply riscv-asm.md:607]: Yes, Github silently added the newline to the last line. Supposedly this is more "POSIX-ly correct". I'm leaving it.

Comment (aswaterman) [reply riscv-asm.md:607]: sgtm

**Comment by aswaterman** [comment]
lgtmf

**Pull request merged**